extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMetaInfoHelper.java

```diff
@@ -171,15 +171,40 @@ private static void getEmergencyOneboxRenderer(
             // usually an encouragement like "We are with you"
             final String title = getTextFromObjectOrThrow(r.getObject("title"), "title");
             // usually a phone number
-            final String action = getTextFromObjectOrThrow(r.getObject("actionText"), "action");
+            final String action;
+            if (r.has("actionText")) {
+                action = getTextFromObjectOrThrow(r.getObject("actionText"), "action");
+            } else if (r.has("contacts")) {
+                final JsonArray contacts = r.getArray("contacts");
+                final StringBuilder stringBuilder = new StringBuilder();
+                int i = 0;
+                final int contactsSize = contacts.size();
+                if (contactsSize != 0) {
+                    // Loop over contacts item from the first contact to the last one, if there is
+                    // not only one, in order to not add an unneeded line return
+                    for (; i < contactsSize - 1; i++) {
+                        stringBuilder.append(getTextFromObjectOrThrow(contacts.getObject(i)
+                                .getObject("actionText"), "contacts.actionText"));
+                        stringBuilder.append("\n");
+                    }
+                    // Add the latest contact without an extra line return
+                    stringBuilder.append(getTextFromObjectOrThrow(contacts.getObject(i)
+                            .getObject("actionText"), "contacts.actionText"));
+                }
+
+                action = stringBuilder.toString();
+            } else {
+                action = null;
+            }
```

Comment:
I would tell you to use streams here, but I guess they don't behave well with exceptions, so let's keep it iterative.

Let's remove the `\n` logic and use an empty string for no action (also requires next suggestion comment)

```suggestion
            } else if (r.has("contacts")) {
                final JsonArray contacts = r.getArray("contacts");
                final StringBuilder stringBuilder = new StringBuilder();

                // Loop over contacts item from the first contact to the last one
                for (int i = 0; i < contacts.size(); i++) {
                    stringBuilder.append("\n");
                    stringBuilder.append(getTextFromObjectOrThrow(contacts.getObject(i)
                            .getObject("actionText"), "contacts.actionText"));
                }

                action = stringBuilder.toString();
            } else {
                action = "";
            }
```